docparse/chunking_strategies.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: 'Chunking Strategies'
+description: 'The chunking strategies supported by Aryn DocParse'
+icon: 'brackets-curly'
+---
+
+
+
+When calling DocParse, you can specify a chunking strategy to the partition_file call. You can enable the default chunking options by specifying an empty dict:
```

Comment:
`partition_file`, `dict` (add `` around types and functions).

Reply:
will do